tasks/modules.py

```diff
@@ -57,6 +57,25 @@ def import_path(self):
             path += "/" + self.path
         return path
 
+    @property
```

Comment:
This is called exactly once 😄, but yeah I'll compute them in the constructor.

Reply:
> Maybe we can compute the dependencies in the constructor instead

This is making life way harder actually 😄, because this means that code is run when any invoke task is run, which makes unrelated invoke tasks uses fail in the CI because they're run from weird places where the `__compute_dependencies` code cannot work, eg. https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/106403621